setup.py

```diff
@@ -22,7 +22,7 @@
         'Programming Language :: Python :: 2.7',
         'Programming Language :: Python :: 3.6',
     ],
-    install_requires=['jinja2', 'setuptools', 'urlextract', 'packaging'],
+    install_requires=['boto3', 'jinja2', 'setuptools', 'urlextract', 'packaging'],
```

Comment:
i can imagine a future world in which we have split out the CodeArtifact functionality into a separate file, and it's packaged as an `extra`, but let me not get ahead of myself